string robot_name
---
int64[] joint_index  # is link_index: https://pybullet.org/Bullet/phpBB3/viewtopic.php?t=12728
string[] joint_name
string info
bool success